packages/kyt-cli/cli/actions/setup.js

```diff
@@ -414,10 +414,25 @@ module.exports = (flags, args) => {
     removeTmpRepo();
 
     if (localPath) {
-      shell.cp('-r', localPath, tmpRepo);
-      afterClone();
+      shell.exec(`cp -R ${localPath} ${tmpRepo}`);
+      afterCopy();
+    } else if (npmName) {
+      shell.mkdir(tmpRepo);
+      shell.cd(tmpRepo);
+      const fakePkgPath = `${tmpRepo}/package.json`;
+      fs.writeFileSync(fakePkgPath, JSON.stringify(fakePackageJson, null, 2));
+      let iCmd = 'npm install';
```

Comment:
just latest for now. I think we want to have people install latest, but we could talk about supplying versions in the future if we see a use case